views/js/app.jsx

```diff
@@ -111,34 +112,45 @@ class LoginHome extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-        reps: []
+      user_guid: null,
+      reps: []
     };
-
-    this.serverRequest = this.serverRequest.bind(this);
-    this.logout = this.logout.bind(this);
   }
 
-  logout() {
+  logout = () => {
     localStorage.removeItem("id_token");
     localStorage.removeItem("access_token");
     localStorage.removeItem("profile");
     location.reload();
   }
 
-  serverRequest() {
+  serverRequest = () => {
     $.get("http://localhost:3000/api/localreps", res => {
       this.setState({
-        reps: res
+        user_guid: res.user_guid,
+        reps: res.users_rep_list
       });
     });
   }
 
+  deleteInParent = (guid) => {
+    // TODO: I don't think we need to make a network call here, but this depends on how we want to store a user's
+    // modified reps - in a database or just in local state
+    // $.post(`http://localhost:3000/api/localreps/edit?editTask=remove&user_guid=55ee03f2dcd8c8e46b91cbb2e70d9e&rep_guid=${guid}`, res => {
+    //    return res;
+    // });
```

Comment:
Included this comment for discussion on how we want to store a user's reps in the long-term, but we can remove it before merging!

Reply:
We likely won't need a network call, because we will push the action through kafka as well..